source/features/mark-private-orgs.tsx

```diff
@@ -22,9 +22,10 @@ async function init(): Promise<false | void> {
 		return false;
 	}
 
-	const publicOrganizations = await getPublicOrganizations(getUsername());
+	const publicOrganizationsNames = await getPublicOrganizationsNames(getUsername());
 	for (const org of orgs) {
-		if (!publicOrganizations.includes(org.pathname.slice(1))) {
+		// https://github.com/sindresorhus/refined-github/issues/2830#issuecomment-591111897
+		if (!publicOrganizationsNames.includes(org.pathname.slice(1).replace('organizations/', ''))) {
```

Comment:
Either drop slice or replace. They can do the same thing. `replace(/^[/]organizations[/]/, '')` has clear intent

Reply:
Good idea! I have incorporated your feedback via 6e6af3a 🙏